iceoryx_utils/test/moduletests/test_posix_mutex.cpp

```diff
@@ -80,15 +80,15 @@ TEST_F(Mutex_test, DestructorFailsOnLockedMutex)
             std::thread* t;
             {
                 iox::posix::mutex mtx{false};
-                iox::posix::Timer hold(1000_ms);
+                iox::clock::DeadlineTimer hold(1000_ms);
                 t = new std::thread([&] {
                     mtx.lock();
-                    iox::posix::Timer ct(5000_ms);
-                    while (!ct.hasExpiredComparedToCreationTime()) // come back in any case!
+                    iox::clock::DeadlineTimer ct(5000_ms);
+                    while (!ct.hasExpired()) // come back in any case!
```

Comment:
@elBoberido  Not really part of the timer change, but why no ``sleep_for`` here? Is the spinning wait essential? I understand that a sleep_for may yield the thread execution (after a short spin, presumably).

Reply:
Can i create a separate story for this to track and update this test?

Reply:
sure

Reply:
https://github.com/eclipse-iceoryx/iceoryx/issues/484 created a separate issue https://github.com/eclipse-iceoryx/iceoryx/issues/484